src/styles/global.css

```
@@ -147,6 +146,9 @@ h3 {
   overflow: initial;
 }
 
+/* Import inline code styles after Gatsby/Prism rules to ensure proper specificity */
+@import "./inline-code.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)